src/System.Management.Automation/engine/parser/tokenizer.cs

```diff
@@ -2029,7 +2136,12 @@ private TokenFlags ScanStringExpandable(StringBuilder sb, StringBuilder formatSb
                     if (c1 != 0)
                     {
                         SkipChar();
-                        c = Backtick(c1);
+                        string str = Backtick(c1);
+                        if (IsSurrogatePair(str, ref c, sb, formatSb))
+                        {
+                            // The character has been processed and appended to the string builders
```

Comment:
Maybe if the method name was something other than `Is..` - then it might be obvious, but I asked for the comment to make it extra clear what processing was done so that if someone added new code that gets skipped, it would be more obvious that `IsSurrogatePair` might need additional work.

---
In reply to: [123819510](https://github.com/PowerShell/PowerShell/pull/3958#discussion_r123819510) [](ancestors = 123819510)